cpp/example_code/rds/tests/mock_input/16-DescribeDBInstances.xml

```diff
@@ -0,0 +1,129 @@
+<DescribeDBInstancesResponse xmlns="http://rds.amazonaws.com/doc/2014-10-31/">
+    <DescribeDBInstancesResult>
+        <DBInstances>
+            <DBInstance>
+                <AllocatedStorage>5</AllocatedStorage>
+                <AssociatedRoles/>
+                <DBParameterGroups>
+                    <DBParameterGroup>
+                        <DBParameterGroupName>doc-example-parameter-group</DBParameterGroupName>
+                        <ParameterApplyStatus>in-sync</ParameterApplyStatus>
+                    </DBParameterGroup>
+                </DBParameterGroups>
+                <DBSecurityGroups/>
+                <EngineVersion>5.7.33</EngineVersion>
+                <MasterUsername>foo</MasterUsername>
+                <CertificateDetails>
+                    <CAIdentifier>rds-ca-2019</CAIdentifier>
+                </CertificateDetails>
+                <DBInstanceClass>db.t2.micro</DBInstanceClass>
+                <StorageThroughput>0</StorageThroughput>
+                <HttpEndpointEnabled>false</HttpEndpointEnabled>
+                <ReadReplicaDBInstanceIdentifiers/>
+                <CustomerOwnedIpEnabled>false</CustomerOwnedIpEnabled>
+                <MonitoringInterval>0</MonitoringInterval>
+                <DBInstanceStatus>creating</DBInstanceStatus>
+                <BackupRetentionPeriod>1</BackupRetentionPeriod>
+                <OptionGroupMemberships>
+                    <OptionGroupMembership>
+                        <OptionGroupName>default:mysql-5-7</OptionGroupName>
+                        <Status>in-sync</Status>
+                    </OptionGroupMembership>
+                </OptionGroupMemberships>
+                <BackupTarget>region</BackupTarget>
+                <CACertificateIdentifier>rds-ca-2019</CACertificateIdentifier>
+                <DbInstancePort>0</DbInstancePort>
+                <DbiResourceId>db-CKGCM6BVK67HH7GSPRBYWX5DSU</DbiResourceId>
+                <PreferredBackupWindow>03:30-04:00</PreferredBackupWindow>
+                <DeletionProtection>false</DeletionProtection>
+                <DBInstanceIdentifier>doc-example-instance</DBInstanceIdentifier>
+                <DBInstanceArn>arn:aws:rds:us-east-1:123502194722:db:doc-example-instance</DBInstanceArn>
```

Comment:
Do the mocks include real or mocked arns and ids? They seem to include a lot, is all of it necessary for mocking?

Reply:
I am not sure what is necessary. I just now deleted some things and broke the tests. 
Also, what works now might break in the future. I do not like to make assumptions about what the SDK might reference.

I am replacing the account ID in the ARN, which is mine, to 1234.... I think that is the only thing that needs to be anonymized.